### PR TITLE
chore: update dictionary/set method names

### LINKED ISSUE #52: Rename dictionary and set read/write functions

### PR DESCRIPTION
**Pull request by poppoerika**
Updated dictionary/set methods names in `SimpleCacheClient` as directed in #52.
Also, updated the names in the `CacheClientTest` accordingly.

Closes #52

### CODE REVIEW

**Review by cprice404 — DISMISSED** [review]
lgtm!  let's have @pgautier404  take a quick skim too and then 🚢

File: src/Cache/SimpleCacheClient.php [hunk 1]
```diff
@@ -8,12 +8,14 @@
 use Momento\Cache\CacheOperationTypes\CacheDictionaryDeleteResponse;
 use Momento\Cache\CacheOperationTypes\CacheDictionaryFetchResponse;
 use Momento\Cache\CacheOperationTypes\CacheDictionaryGetBatchResponse;
```
Comment (pgautier404) [inline src/Cache/SimpleCacheClient.php:10]: Please remove `CacheDictionaryGetBatchResponse`.

File: src/Cache/SimpleCacheClient.php [hunk 2]
```diff
@@ -8,12 +8,14 @@
...
-use Momento\Cache\CacheOperationTypes\CacheDictionaryGetResponse;
+use Momento\Cache\CacheOperationTypes\CacheDictionaryGetFieldResponse;
+use Momento\Cache\CacheOperationTypes\CacheDictionaryGetFieldsResponse;
 use Momento\Cache\CacheOperationTypes\CacheDictionaryIncrementResponse;
 use Momento\Cache\CacheOperationTypes\CacheDictionaryRemoveFieldResponse;
 use Momento\Cache\CacheOperationTypes\CacheDictionaryRemoveFieldsResponse;
 use Momento\Cache\CacheOperationTypes\CacheDictionarySetBatchResponse;
```
Comment (pgautier404) [inline src/Cache/SimpleCacheClient.php:16]: Please remove `CacheDictionarySetBatchResponse`.

File: src/Cache/SimpleCacheClient.php [hunk 3]
```diff
@@ -22,6 +24,7 @@
 use Momento\Cache\CacheOperationTypes\CacheListPushBackResponse;
 use Momento\Cache\CacheOperationTypes\CacheListPushFrontResponse;
 use Momento\Cache\CacheOperationTypes\CacheListRemoveValueResponse;
+use Momento\Cache\CacheOperationTypes\CacheSetAddElementResponse;
 use Momento\Cache\CacheOperationTypes\CacheSetAddResponse;
```
Comment (pgautier404) [inline src/Cache/SimpleCacheClient.php:28]: Please remove `CacheSetAddResponse`.

File: src/Cache/_ScsDataClient.php [hunk 4]
```diff
@@ -430,7 +430,7 @@ public function listErase(string $cacheName, string $listName, ?int $beginIndex
         return new CacheListEraseResponseSuccess();
     }
 
-    public function dictionarySet(string $cacheName, string $dictionaryName, string $field, string $value, bool $refreshTtl, ?int $ttlSeconds = null): CacheDictionarySetResponse
+    public function dictionarySet(string $cacheName, string $dictionaryName, string $field, string $value, bool $refreshTtl, ?int $ttlSeconds = null): CacheDictionarySetFieldResponse
```
Comment (pgautier404) [inline src/Cache/_ScsDataClient.php:433]: I'm a little concerned about how these method names no longer match the SCC methods. Having `$this->client->dictionarySetField()` end up calling `dictionarySet()` here seems like it's going to be very confusing for us to maintain in the future. I'd propose we go ahead and propagate the function name changes into the data client as well. @cprice404 do you have a strong opinion here?

Comment (poppoerika) [reply src/Cache/_ScsDataClient.php:433]: I had the same question there.

Comment (pgautier404) [reply src/Cache/_ScsDataClient.php:433]: Let's please go ahead and update them.

Comment (cprice404) [reply src/Cache/_ScsDataClient.php:433]: your suggestion makes sense to me

Comment (poppoerika) [reply src/Cache/_ScsDataClient.php:433]: > Let's please go ahead and update them.

Roger.

**Review by pgautier404 — APPROVED** [review]
LGTM!

**Pull request merged**